tests/test_transformations.py

```diff
@@ -9,9 +9,13 @@
 import pandas as pd
 import pytest
 import xarray as xr
+from pandas import TimedeltaIndex as TDI
+from pandas import Timestamp as TS
+from pandas import date_range
 
 import weldx.transformations as tf
 import weldx.utility as ut
+from weldx.constants import WELDX_QUANTITY as Q_
```

Comment:
```suggestion
from weldx import Q_
```
let's make this the default :)